python/pyproject.toml

```
@@ -48,7 +48,7 @@ build-backend = "maturin"
 
 [project.optional-dependencies]
 tests = [
```

Let's leave a comment reminding us why we made this bound:
```suggestion
tests = [
    # Can't use pandas 2.1 in tests until this duckdb bug is fixed:
    # https://github.com/duckdb/duckdb/issues/8735
```